lib/log.bash

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+# Avoid duplicate inclusion
+if [[ "${__bi_log_imported:-}" == "loaded" ]]; then
```

Comment:
Thinking on it, this can likely just be a `-n` check as I don't know if the _actual_ value conveys any meaning?

Reply:
Done 👍